test/test_testing.py

```diff
@@ -85,6 +85,7 @@ def test(tmp_path):
             # mac/linux.
             "CIBW_TEST_COMMAND": "false || pytest {project}/test",
             "CIBW_TEST_COMMAND_WINDOWS": "COLOR 00 || pytest {project}/test",
+            "_PYODIDE_EXTRA_MOUNTS": str(tmp_path),
```

Comment:
If adding this is always necessary, can we add `"_PYODIDE_EXTRA_MOUNTS": str(project_dir)` to the actual code, rather than adding it each time in the tests? It's possible that users might want to build from a temp dir too sometimes, and cibulidwheel itself builds from temp dirs when building from an sdist I think.

Reply:
I will try that. I think we could add it only if `str(project_dir).startswith("/tmp")`. 

Another option I have been considering is just mounting `/tmp/*` in the command line runner. Or perhaps the best thing would be to locate and fix bug in the nodefs implementation that prevents us from mounting `/tmp` itself...